controllers/message_answer.go

```diff
@@ -15,15 +15,112 @@
 package controllers
 
 import (
+	"encoding/json"
 	"fmt"
 	"strings"
 
 	"github.com/beego/beego"
+	"github.com/beego/beego/logs"
 	"github.com/casibase/casibase/model"
 	"github.com/casibase/casibase/object"
 	"github.com/casibase/casibase/util"
 )
 
+// SetChatPrompt
+// @Title SetChatPrompt
+// @Tag Chat API
+// @Description set custom prompt for a chat
+// @Param id query string true "The id of chat"
+// @Param prompt body string true "The custom prompt"
+// @Success 200 {object} controllers.Response "Success"
+// @router /set-chat-prompt [post]
+func (c *ApiController) SetChatPrompt() {
```

Comment:
Don't need to add new API, see how this UI widget works:

![image](https://github.com/user-attachments/assets/fc5dab08-f00a-4a62-9533-f61a767cae55)